blueprints/vendor-shim/files/vendor/__name__-shim.js

```diff
@@ -0,0 +1,9 @@
+(function() {
```

Comment:
@stefanpenner is this file named properly for the following to work?

```bash
# we want to generate `vendor/moment-shim.js`
$ ember generate vendor-shim moment
```

Additionally, is there an API to add `app.import('vendor/moment-shim.js')` to the `ember-cli-build.js`?